src/apps/llamapay/llamapay.definition.ts

```diff
@@ -0,0 +1,43 @@
+import { Register } from '~app-toolkit/decorators';
+import { appDefinition, AppDefinition } from '~app/app.definition';
+import { AppAction, AppTag, GroupType } from '~app/app.interface';
+import { Network } from '~types/network.interface';
+
+export const LLAMAPAY_DEFINITION = appDefinition({
+  id: 'llamapay',
+  name: 'Llamapay',
+  description: 'LlamaPay integration for Zapper',
```

Comment:
`LlamaPay is a multi-chain protocol that allows you to automate transactions and stream them by the second.` would be better!